### PR TITLE
Fix some places which doesn't repsect org full name setting

### LINKED ISSUE #31345: Italian alphabet onlyin repo + org names
### Description

Apparently repo and organizations names cant have

ñ, ç, ț, ă, â, ș, î

yes "Gobierno de España" the Spanish government is not allowed to be an organization 

image

dont ask to have ñ->n, last time White House did that i went from feliz año to feliz ano 

### Gitea Version

1,22

### Can you reproduce the bug on the Gitea demo site?

Yes

### Log Gist

_No response_

### Screenshots

image

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

container

### Database

None

### PR DESCRIPTION
**Pull request by lunny**
Partially fix #31345

### CODE REVIEW

File: web_src/js/components/DashboardRepoList.vue [hunk 1]
```diff
@@ -471,7 +472,7 @@ export default sfc; // activate the IDE's Vue plugin
           <li class="tw-flex tw-items-center tw-py-2" v-for="org in organizations" :key="org.name">
             <a class="repo-list-link muted" :href="subUrl + '/' + encodeURIComponent(org.name)">
               <svg-icon name="octicon-organization" :size="16" class-name="repo-list-icon"/>
-              <div class="text truncate">{{ org.name }}</div>
+              <div class="text truncate">{{ defaultShowFullName && org.full_name ? org.full_name : org.name }}</div>
```
Comment (wxiaoguang) [inline web_src/js/components/DashboardRepoList.vue]: It should be ``org.full_name ? `${org.full_name} (${org.name})` : org.name``

No need to introduce `defaultShowFullName` which doesn't seem to make sense.

File: templates/admin/org/list.tmpl [hunk 2]
```diff
@@ -52,7 +52,7 @@
 						<tr>
 							<td>{{.ID}}</td>
 							<td>
-								<a href="{{.HomeLink}}">{{.Name}}</a>
+								<a href="{{.HomeLink}}">{{if and DefaultShowFullName .FullName}}{{.FullName}}({{.Name}}){{else}}{{.Name}}{{end}}</a>
```
Comment (wxiaoguang) [inline templates/admin/org/list.tmpl]: Space is missing

Comment (lunny) [reply templates/admin/org/list.tmpl]: Which space did you mean?

File: web_src/js/components/DashboardRepoList.vue (hunk 1 above)
Comment (lunny) [reply web_src/js/components/DashboardRepoList.vue]: a60fce5aa8c6f96f44519a424409480c2babaf60

File: templates/admin/org/list.tmpl (hunk 2 above)
Comment (wxiaoguang) [reply templates/admin/org/list.tmpl]: Do you write sentences like "something (something else)" or "something(something else)"?

Comment (lunny) [reply templates/admin/org/list.tmpl]: d8c9044637

**Review by wxiaoguang — APPROVED** [review]

**Review by Zettat123 — APPROVED** [review]

**Pull request merged**